lib/rails_event_store/client.rb

```diff
@@ -1,7 +1,7 @@
 module RailsEventStore
   class Client
 
-    def initialize(repository: RailsEventStoreActiveRecord::EventRepository.new,
```

Comment:
I understand the idea. How would you handle the fact that, if i want to use RES with MongoDB, I don't want to install AR dependencies?

We could improve the following PR by having each dependencies taking care of the ORM (RES_AR, RES_MONGO, ...) setting the proper adapter, event repository and any other requirements when the gem loads.